### PR TITLE
[KOGITO-3446] - Remove KAFKA_BOOTSTRAP_SERVERS env check from Jobs se…

### PR DESCRIPTION
**Pull request by vaibhavjainwiz**
…rvice scripts

jira issue : https://issues.redhat.com/browse/KOGITO-3446

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors guide](README.md#contributing-to-kogito-images-repository)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Your feature/bug fix has a testcase that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: tests/features/kogito-jobs-service.feature [hunk 1]
```diff
@@ -30,21 +30,14 @@ Feature: Kogito-jobs-service feature.
...
-      | variable      | value |
-      | ENABLE_EVENTS | true  |
-    Then container log should contain KAFKA_BOOTSTRAP_SERVERS env not found, please set it.
-
   Scenario: verify if the events is correctly enabled
     When container is started with env
       | variable                | value           |
       | SCRIPT_DEBUG            | true            |
       | ENABLE_EVENTS           | true            |
-      | KAFKA_BOOTSTRAP_SERVERS | localhost:11111 |
```
Comment (ricardozanini) [inline tests/features/kogito-jobs-service.feature:44]: Maybe adding `KOGITO_JOBS_PROPS=-Dquarkus.kafka.bootstrap-servers=localhost:11111` just to make sure that the Operator set up will work.

File: tests/features/kogito-jobs-service.feature [hunk 2]
```diff
@@ -30,21 +30,14 @@ Feature: Kogito-jobs-service feature.
...
-    When container is started with env
-      | variable      | value |
-      | ENABLE_EVENTS | true  |
-    Then container log should contain KAFKA_BOOTSTRAP_SERVERS env not found, please set it.
-
   Scenario: verify if the events is correctly enabled
     When container is started with env
       | variable                | value           |
       | SCRIPT_DEBUG            | true            |
       | ENABLE_EVENTS           | true            |
```
Comment (ricardozanini) [inline tests/features/kogito-jobs-service.feature]:
```suggestion
      | ENABLE_EVENTS           | true            |
      | KOGITO_JOBS_PROPS | -Dquarkus.kafka.bootstrap-servers=localhost:11111 |
```

**Review by ricardozanini — CHANGES_REQUESTED** [review]
Thanks @vaibhavjainwiz ! I'd say to just try to pass the same property that will be set by the operator to make sure everything will work.

File: tests/features/kogito-jobs-service.feature (hunk 1 above)
Comment (vaibhavjainwiz) [reply tests/features/kogito-jobs-service.feature:44]: done

File: tests/features/kogito-jobs-service.feature (hunk 2 above)
Comment (vaibhavjainwiz) [reply tests/features/kogito-jobs-service.feature]: done

**Review by ricardozanini — APPROVED** [review]

**Comment by spolti** [comment]
@vaibhavjainwiz did you check the erros on CI?

**Review by sutaakar — APPROVED** [review]

**Review by radtriste — APPROVED** [review]

**Review by Kaitou786 — APPROVED** [review]

**Comment by ricardozanini** [comment]
I believe we need to verify the CI before merging this one. @vaibhavjainwiz do you have any clues about the problem?

**Comment by spolti** [comment]
Here is the problem: https://issues.redhat.com/browse/KOGITO-3469
got the same issue locally.

**Review by spolti — APPROVED** [review]

**Comment by spolti** [comment]
merging, will disable this tests while it is not fixed in another pull request.

**Pull request merged**